test/test_achievements_by_person.py

```diff
@@ -1,20 +1,16 @@
 from . import BaseTestCase
 
+
 class TestAchievementsByPerson(BaseTestCase):
-	def test_get_achievements_by_person(self):
-		expected = dict(achievements = [
-							{
-								'age': 21, 
-								'description': 'Founded Apple Computer Company with Wozniak'
-							}, 
-							{
-								'age': 50, 
-								'description': 'Stanford speech'
-							}])
-		response = self.client.get('/achievements/Steve_Jobs/')
-		self.assertItemsEqual(response.json['achievements'], expected['achievements'])
-		self.assertTrue(response.status_code == 200)
+    def test_get_achievements_by_person(self):
+        expected = {'achievements': [{'age': 21,
```

Comment:
@matejkrajcovic wouldn't it be better to just load this from a json file? I know, fast tests and everything, but any addition you make here will just clutter this test which I'd argue is a good reason to put the output in a file which could get loaded at this line into `expected`.